### PR TITLE
with debugger

### PR DESCRIPTION
**Pull request by forki**
/cc @MangelMaxime

### CODE REVIEW

**Comment by MangelMaxime** [comment]
Yes should be working

**Comment by forki** [comment]
oh it's red. any idea what's broken with webpack?

File: paket.lock [hunk 1]
```diff
@@ -1,17 +1,25 @@
 STORAGE: NONE
 NUGET
   remote: https://www.nuget.org/api/v2
-    dotnet-fable (2.0.2) - clitool: true
+    dotnet-fable (1.3.19) - clitool: true
```
Comment (MangelMaxime) [inline paket.lock]: @forki This version is incorrect you are back to Fable 1

Comment (forki) [reply paket.lock]: urgs

**Pull request merged**